tests/rptest/tests/enterprise_features_license_test.py

```diff
@@ -42,12 +43,26 @@ class Features(IntEnum):
     leadership_pinning = 10
 
 
+# TODO(oren): extend to multiple configs?
+FEATURE_DEPENDENT_CONFIG = {
+    Feature.audit_logging: 'audit_enabled',
+    Feature.cloud_storage: 'cloud_storage_enabled',
+    Feature.partition_auto_balancing_continuous:
+    'partition_autobalancing_mode',
+    Feature.core_balancing_continuous: 'core_balancing_continuous',
+    Feature.gssapi: 'sasl_mechanisms',
+    Feature.oidc: 'sasl_mechanisms',
+    Feature.schema_id_validation: 'enable_schema_id_validation',
+    Feature.datalake_iceberg: 'datalake_iceberg',
+    Feature.leadership_pinning: 'default_leaders_preference',
+}
+
 SKIP_FEATURES = [
-    Features.audit_logging,  # NOTE(oren): omit due to shutdown issues
-    Features.
+    Feature.audit_logging,  # NOTE(oren): omit due to shutdown issues
```

Comment:
maybe this is fixed thanks to your changes made in the audit subsystem?  But you don't have to check/fix here

Reply:
I tried, it isn't 😔